app/docs/md/conventions/components.md

```diff
@@ -114,6 +129,29 @@ If you have an existing Enhance Element you can always import it into your Compo
 
 </doc-callout>
 
+### Making Components available in the browser
+
+Now that we have written a client side component we need to register it with the browser at runtime. To do this we will create a new file `app/browser/index.mjs`. This is the file where we will import all of the components that will be rendered client-side. Using our `MyCard` component as an example our `app/browser/index.mjs` would look like this:
+
+```javascript
+// app/browser/index.mjs
+import MyCard from "../components/my-card.mjs"
+export { MyCard }
+```
+
+This will create a file called `public/browser/index.mjs` that you include as a `script` tag on any pages you want to use the `MyCard` component. For example:
```

Comment:
```suggestion
This file will be mirrored and made available to your application as `public/browser/index.mjs`. You can then use this file as a `script` tag’s source on any pages in which you want to use the `MyCard` component. For example:
```

Reply:
Is mirroring the right wording here? It's actually rolling up all the references in that file and bundling them into the public/browser folder.

Reply:
Hmm yeah mirroring isn't correct then. Maybe… this?

```suggestion
This will create a new client side bundle, available at `public/browser/index.mjs`. You can use this file as a `script` tag’s source on any pages in which you want to use the `MyCard` component. For example:
```